QUARKUS-743.md

```diff
@@ -0,0 +1,70 @@
+# QUARKUS-743 - Verify Code Starters
+
+JIRA link: https://issues.redhat.com/browse/QUARKUS-743
+
+The goal is to verify the code starters available in some Quarkus extensions.
+
+For native mode similar test coverage as for JVM mode needs to be ensured.
+
+## Scope of the testing
+
+- For Quarkus 1.11, the Quarkus extensions that provide a code starter are:
+
+```
+quarkus-amazon-lambda
+quarkus-azure-functions-http
+quarkus-config-yaml
+quarkus-funqy-amazon-lambda
+quarkus-funqy-http
+quarkus-funqy-knative-events
+quarkus-google-cloud-functions
+quarkus-google-cloud-functions-http
+quarkus-logging-json
+quarkus-picocli
+quarkus-qute
+quarkus-resteasy
+quarkus-resteasy-jackson
+quarkus-resteasy-reactive
+quarkus-spring-web
+```
+
+The list has been provided by running: `curl https://code.quarkus.io/api/extensions 2>/dev/null | jq '.[] | select(.tags[] | contains("provides-example")) | .id' | sort | uniq`.
+
+For each Quarkus extension with code starter:
+ - Verify that the Quarkus extensions defined in the `pom.xml` are marked with the `provides-example` tag in `code.quarkus.io`. 
+ - Verify that the code start compile.
+ - Verify that the code start is fully working.
+
+Following actions were taken to ensure test coverage and automation for native image:
+ - Define the scope of the testing
+ - Analyze the possible impact on Quarkus itself - e.g performance, compatibility
+ - Determine impact on test suites - e.g. new test suite, increased complexity of existing test suite, maintenance burden
+
+ As part of the development, code starters are already being verified using `Maven` and `Gradle` in [these tests](https://github.com/quarkusio/quarkus/blob/master/integration-tests/devtools/src/test/java/io/quarkus/devtools/codestarts/quarkus/QuarkusCodestartRunIT.java#L74-L90). 
+
+### Impact on testsuites and testing automation:
+ - Test development in Start-Stop testsuite in order to cover:
+   - Verify that the code start compile.
```

Comment:
compiles || runs in dev mode || compiles + runs java -jar
Which one :) ?

Reply:
I've changed this line to make it clear that it should work either DEV or JAV modes.